deploy/docker/entrypoint.sh

```diff
@@ -168,7 +168,7 @@ init_replica_set() {
     # Check mongodb cloud Replica Set
     echo "Checking Replica Set of external MongoDB"
 
-    if appsmithctl check_replica_set; then
+    if [[ $(mongo $APPSMITH_MONGODB_URI --quiet --eval "rs.status().ok") -eq 1 ]]; then
```

Comment:
Let's wrap the URL in double-quotes. If it contains any special characters in its password, for example, it'll still work. :)

```suggestion
    if [[ $(mongo "$APPSMITH_MONGODB_URI" --quiet --eval "rs.status().ok") -eq 1 ]]; then
```